api-inference-community/docker_images/common/tests/test_api_text_to_image.py

```diff
@@ -0,0 +1,67 @@
+import base64
+import os
+from io import BytesIO
+from unittest import TestCase, skipIf
+
+import PIL
+from app.main import ALLOWED_TASKS
+from starlette.testclient import TestClient
+from tests.test_api import TESTABLE_MODELS
+
+
+@skipIf(
+    "text-to-image" not in ALLOWED_TASKS,
+    "text-to-image not implemented",
+)
+class TextToImageTestCase(TestCase):
+    def setUp(self):
+        model_id = TESTABLE_MODELS["text-to-speech"]
+        self.old_model_id = os.getenv("MODEL_ID")
```

Comment:
not important, but i don't quite understand why there is a `model_id` and an `old_model_id`: what does the "old" refer to?

Reply:
This part is just copied from other tests. If I recall correctly, this is to ensure that when you run `pytest ...` you don't override environment variables in your computer. Does that make sense?

Reply:
ah ok thanks for the context!